server/.classpath

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<classpath>
-	<classpathentry kind="src" path="src"/>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
-	<classpathentry kind="lib" path="lib/servlet-api-2.5.jar" sourcepath="lib-src/servlet-api-2.5-sources.jar"/>
-	<classpathentry kind="lib" path="lib/jetty-6.1.22.jar" sourcepath="lib-src/jetty-6.1.22-sources.jar"/>
-	<classpathentry kind="lib" path="lib/jetty-util-6.1.22.jar" sourcepath="lib-src/jetty-util-6.1.22-sources.jar"/>
-	<classpathentry kind="lib" path="lib/jdatapath-alpha2.jar" sourcepath="lib-src/jdatapath-alpha2-sources.jar"/>
-	<classpathentry kind="lib" path="lib/slf4j-api-1.5.6.jar" />
-	<classpathentry kind="lib" path="lib/log4j-1.2.15.jar" />
-	<classpathentry kind="lib" path="lib/slf4j-log4j12-1.5.6.jar" />
-	<classpathentry kind="output" path="classes"/>
-</classpath>
```

Comment:
I think this file should be kept too

Reply:
Current OpenRefine is a single project and single classpath. It is not worth keep it if we don't use it and have to maintain it and confuse programmer. Once we move to other build system such as gradle or maven, those setting will be all gone. Since we don't use it, deleting them can pave the road while we do the gradle/maven migration.

Reply:
okay, fine for this time